src/aiperf/timing/branch_orchestrator.py

```
@@ -0,0 +1,1397 @@
+        None when no timing evidence exists (e.g. --ignore-trace-delays
+        datasets), which disables offsets for the branch.
+        """
+        start = _as_timestamp_ms(getattr(branch, "start_timestamp_ms", None))
+        if start is not None:
+            return start
+        child_starts: list[float] = []
+        for child_id in branch.child_conversation_ids:
+            try:
+                meta = self._cs.get_metadata(child_id)
+            except Exception:  # noqa: BLE001 - missing metadata = no timing evidence
+                continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)